classes/controller/FrontController.php

```diff
@@ -825,23 +825,23 @@ protected function geolocationManagement($defaultCountry)
      */
     public function setMedia()
     {
-        $this->registerStylesheet('theme-main', '/assets/css/theme.css', 'all', 0);
-        $this->registerStylesheet('theme-custom', '/assets/css/custom.css', 'all', 1000);
+        $this->registerStylesheet('theme-main', '/assets/css/theme.css', ['media' => 'all', 'priority'=> 0]);
```

Comment:
I don't know. We're meant to follow Symfony standards but they removed this requirement. So I assume it applies to us as well.

(and let's be honest, we all like `[]` haha)